_opensearch/query-dsl/geo-and-xy/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Geographic and xy queries
+parent: Query DSL
+has_children: true
+nav_order: 50
+---
+
+# Geographic and xy queries
+
+Geographic and xy queries let you search fields that contain points and shapes on a map or coordinate plane. Geoshape and xy queries are similar, but the former searches [geographic fields] while the latter searches Cartesian fields.
```

Comment:
Geoshape --> geographic?

Reply:
Why the brackets around geographic fields?

Reply:
Added a link. Geoshape queries are a type of geographic queries, which is very similar to xy queries. Not all geographic queries are similar to xy.